docs/path-resolution.rst

```diff
@@ -71,8 +71,10 @@ The initial content of the VFS depends on how you invoke the compiler:
 
        solc contract.sol /usr/local/dapp-bin/token.sol
 
-   The source unit name of a file loaded this way is simply the specified path after shell expansion
-   and with platform-specific separators converted to forward slashes.
+   The source unit name of a file loaded this way is constructed by converting its path to a
+   canonical form and making it relative to base path if it is located inside.
```

Comment:
```suggestion
   canonical form and making it relative to the base path if it is located inside.
```

Reply:
Looks like I have some kind of blind spot for this :) No problem with `the path` but in `the base path` the `the` sounds redundant (even though I know it's just fine grammatically) so I keep dropping it.

Reply:
At some point I heard that people with a slavic native language often make such mistakes since there is no such article in these languages. I fear I maybe take this into account when reviewing your PRs and over-correct and see mistakes where there are none. Apologies if this is the case.

Reply:
No worries, it was actually a mistake on my part. Don't hesitate to correct me if I get these wrong :) It's helpful

> At some point I heard that people with a slavic native language often make such mistakes since there is no such article in these languages.

There might be something to it though I think I have English articles internalized at this point. I do not have to think about them and certain phrases just "feel wrong" if the article is wrong or missing. I think it might more that I'm mostly exposed to English on the internet and lots of it is very casual so I tend to drop the articles too much. Often it still sounds fine to me without them and I only realize when someone points it out.